app/controllers/leader_sessions_controller.rb

```diff
@@ -0,0 +1,30 @@
+class LeaderSessionsController < ApplicationController
+  skip_before_action(:force_leader_sign_in, { :only => [:registration_form, :session_form, :add_cookie] })
+
+  def session_form
+    render({ :template => "leader_sessions/sign_in.html.erb" })
+  end
+
+  def registration_form
```

Comment:
Hmm, does it feel sort of like `registration_form` should be in the `leaders_controller`?

Reply:
Yeah, I guess so since it's essentially a `new_form`. The name "registration" just feels tied more to a session but I guess it's not really.

Reply:
@jelaniwoods Yes, I'm borrowing Devise terminology — it uses `new_[MODEL]_registration_path`, `edit_[MODEL]_registration_path`, `new_[MODEL]_session_path`, etc.

If you think there are better, clearer names for the URLs and/or actions, then please feel free. We don't have to correspond with Devise, clarity is paramount.